packages/SidePanel/tests/SidePanel.spec.js

```diff
@@ -88,15 +87,96 @@ describe("SidePanel", () => {
     });
   });
 
+  describe("SidePanel.Group", () => {
+    it("should render with multiple sidepanels", () => {
+      const { getAllByTestId } = renderReactTestingLibrary(
+        <SidePanel.Group>
+          <SidePanel onClose={noop} isOpen>
+            <SidePanel.Header>With Header</SidePanel.Header>
+            <SidePanel.Overlay />
+          </SidePanel>
+          <SidePanel onClose={noop} isOpen>
+            <SidePanel.Header>With Header</SidePanel.Header>
+            <SidePanel.Overlay />
+          </SidePanel>
+          <SidePanel onClose={noop} isOpen>
+            <SidePanel.Header kind="primary">With Header</SidePanel.Header>
+            <SidePanel.Overlay />
+          </SidePanel>
+        </SidePanel.Group>
+      );
+      // data-pka-anchor
+      expect(getAllByTestId("heading")).toHaveLength(3);
+    });
+
+    it("throws the error when sidepanel has only one panel", () => {
```

Comment:
"throws an error when there is only one side panel in a group"

Reply:
done